tools/windows-node-installer/test/e2e/common.go

```diff
@@ -16,6 +16,8 @@ import (
 const (
 	// winRMPort is the secure WinRM port
 	winRMPort = "5986"
+	// sshPort is the SSH port
+	sshPort = "22"
```

Comment:
nit: If you made `sshPort` public in the code you could have directly imported?

Reply:
I think the constants are not being shared across the main packages and the test code.